File: introduction/ITP1_2_B-Range.scala
object Main extends App {
  val sc = new java.util.Scanner(System.in)
  val line = sc.nextLine()

  val str = line.split(" ")
  val a = str(0).toInt
  val b = str(1).toInt
  val c = str(2).toInt

  if (a < b && b < c) {
    println("Yes")
  } else {
    println("No")
  }
}

File: introduction/ITP1_2_B-Range.java
import java.util.*;

public class Main {
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    String[] str = sc.nextLine().split(" ");

    int a = Integer.parseInt(str[0]);
    int b = Integer.parseInt(str[1]);
    int c = Integer.parseInt(str[2]);

    if (a < b && b < c) {
      System.out.println("Yes");
    } else {
      System.out.println("No");
    }
  }
}
